exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -399,8 +413,9 @@ func (ku *Kucoin) UpdateTickers(ctx context.Context, assetType asset.Item) error
 		}
 		for t := range ticks.Tickers {
 			pair, err := currency.NewPairFromString(ticks.Tickers[t].Symbol)
-			if err != nil {
-				return err
+			if err != nil && !errors.Is(err, currency.ErrPairNotFound) {
+				errs = common.AppendError(errs, err)
+				continue
```

Comment:
This is a critical function that should not error, better to return this straight away.

Reply:
amended